### PR TITLE
Create tech component

### PR DESCRIPTION
**Pull request by tebanwolox**
## Summary
- Create tech component
- Create tech.scss
- Add properties in size.scss

## Screenshot
Screen Shot 2019-05-14 at 16 34 55

## Trello Card
https://trello.com/c/9Tlklt5i/22-tecnolog%C3%ADa

### CODE REVIEW

File: scss/colors.scss [hunk 1]
```diff
@@ -2,3 +2,4 @@ $black: #000;
 $blue: #00ADEE;
 $green: #BED23C;
 $grey: #C4C4C4;
+$white: #fff;
```
Comment (nmartinelli23) [inline scss/colors.scss]: Use upper case for color, please.

File: scss/tech.scss [hunk 2]
```diff
@@ -0,0 +1,45 @@
+.tech { 
+  align-items: center; 
+  background-color: $green; 
+  display: flex;
+  flex-direction: column;
+  padding: $tech-padding ;
```
Comment (nmartinelli23) [inline scss/tech.scss]: Use variables just for colors, no need for padding or height/width.

File: index.html [hunk 3]
```diff
@@ -9,5 +9,11 @@
 </head>
 <body>
   <h2 class="title">Front-End Tools</h2> 
+  <div class="tech">
+    <img class="tech-logo" src="assets/react_icon.svg">
```
Comment (henryzarza) [inline index.html]: You forgot define `alt` attribute

File: scss/size.scss [hunk 4]
```diff
@@ -4,6 +4,14 @@ $title-decoration-height: 90px;
 $title-decoration-width: 70px;
 $title-padding-left: 50px;
 $tech-logo-size: 100px;
+$tech-margin: 5px;
+%tech-container-size {
```
Comment (henryzarza) [inline scss/size.scss]: I think that this isn't necessary because you can define in `.tech` class, so please delete it

File: scss/size.scss [hunk 5]
```diff
@@ -4,6 +4,14 @@ $title-decoration-height: 90px;
...
+  border-radius: 5px;
+  height: 100%;
```
Comment (henryzarza) [inline scss/size.scss]: You don't need this

File: scss/size.scss [hunk 6]
```diff
@@ -4,6 +4,14 @@ $title-decoration-height: 90px;
...
+  max-height: 300px;
```
Comment (henryzarza) [inline scss/size.scss]: Delete this

File: scss/tech.scss [hunk 7]
```diff
@@ -0,0 +1,44 @@
+.tech { 
+  align-items: center; 
+  background-color: $green; 
+  display: flex;
+  flex-direction: column;
+  padding: 30px;
+  position: relative;
+  @extend %tech-container-size;
```
Comment (henryzarza) [inline scss/tech.scss]: Delete this

File: scss/tech.scss [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+
+  .tech-logo {
+    height: $tech-logo-size;
+    margin-bottom: $tech-margin;
+    width: $tech-logo-size;
+  }
+
+  .tech-title {
+    font-size: $text-medium 0;
```
Comment (henryzarza) [inline scss/tech.scss]:
```suggestion
    font-size: $text-medium;
```

File: scss/tech.scss [hunk 9]
```diff
@@ -0,0 +1,44 @@
...
+
+  .tech-text { 
+    font-size: $text-small;
+    justify-content: center;
+    margin-top: $tech-margin;
+    text-align: center;
+  }
+
+  &::before {
+    background-color: $white;
```
Comment (henryzarza) [inline scss/tech.scss]: Delete this

File: scss/tech.scss [hunk 10]
```diff
@@ -0,0 +1,44 @@
...
+    border: 1px solid $green;
+    bottom: 5px;
+    content: "";
+    position: absolute;
+    right: 5px;
+    z-index: -1;
+    @extend %tech-container-size;
```
Comment (henryzarza) [inline scss/tech.scss]: Replace by: `border-radius: 5px; height: 100%; width: 100%`

File: scss/tech.scss [hunk 11]
```diff
@@ -1,44 +1,51 @@
 .tech { 
   align-items: center; 
   background-color: $green; 
+  border-radius: 5px;
   display: flex;
   flex-direction: column;
+  height: 100%;
+  max-height: 300px;
```
Comment (henryzarza) [inline scss/tech.scss]: Delete this

File: scss/tech.scss [hunk 12]
```diff
@@ -1,44 +1,51 @@
 .tech { 
   align-items: center; 
   background-color: $green; 
+  border-radius: 5px;
   display: flex;
   flex-direction: column;
+  height: 100%;
```
Comment (henryzarza) [inline scss/tech.scss]: Delete this

File: scss/tech.scss [hunk 13]
```diff
@@ -1,44 +1,51 @@
...
   }
 
   &::before {
-    background-color: $white;
     border: 1px solid $green;
+    border-radius: 5px;
     bottom: 5px;
     content: "";
+    height: 100%;
+    max-height: 300px;
```
Comment (henryzarza) [inline scss/tech.scss]: Delete this

File: scss/tech.scss [hunk 14]
```diff
@@ -1,44 +1,51 @@
...
+    max-width: 200px;
```
Comment (henryzarza) [inline scss/tech.scss]: Delete this

**Pull request closed without merging**